### ISSUE TITLE
Base table or view not found : 1146 table 'magento2.sequence_creditmemo_2'

### ISSUE THREAD

**Issue by jmminy** [magento/data-migration-tool#573]
Hello, I am having an issue while running php bin/magento migrate:delta --reset -a /magento/vendor/magento/data-migration-tool/etc/opensource-to-opensource/1.9.3.8/config.xml

It says the table sequence_credit_memo is not found and stops proceeding. Ignoring the table does not event work.

I am using Magento 1.9.3.8 and try to migrate datas to Magento 2.2.5

### Steps to reproduce

1. I have installed a fresh copy of 2.2.5 + your tool
2. I have added no data at all
3. I ran the data script which worked perfectly after fixing the errors due to custom modules
4. When it broke, I ran delta and this error occured. Products are correctly imported but not orders and I am locked.

### Additional notes
I am running it on a Plesk server and both sites are on the same DB server.
While modifying the config.xml to ignore the Sales Increment step, the script works but keep on trying the same action for ages : 

[2018-08-09 13:11:35][INFO][mode: delta][stage: delta delivering][step: Log Step]: started
[2018-08-09 13:11:35][INFO][mode: delta][stage: volume check][step: Log Step]: started
100% [============================] Remaining Time: < 1 sec
[2018-08-09 13:11:35][WARNING]: Mismatch of entities in the document: customer_visitor Source: 1056 Destination: 8
[2018-08-09 13:11:35][WARNING]: Volume Check failed
[2018-08-09 13:11:35][INFO][mode: delta][stage: delta delivering][step: OrderGrids Step]: started
[2018-08-09 13:11:35][INFO][mode: delta][stage: volume check][step: OrderGrids Step]: started
100% [============================] Remaining Time: < 1 sec
[2018-08-09 13:11:35][WARNING]: Mismatch of entities in the document: sales_order_grid Source: 26273 Destination: 0
[2018-08-09 13:11:35][WARNING]: Mismatch of entities in the document: sales_invoice_grid Source: 18966 Destination: 0
[2018-08-09 13:11:35][WARNING]: Mismatch of entities in the document: sales_shipment_grid Source: 1206 Destination: 0
[2018-08-09 13:11:35][WARNING]: Volume Check failed
[2018-08-09 13:11:35][INFO][mode: delta][stage: volume check][step: OrderGrids Step]: Migration completed successfully
[2018-08-09 13:11:35][INFO][mode: delta][stage: volume check][step: OrderGrids Step]: Automatic restart in 5 sec. Use CTRL-C to abort^C
$ php -d memory_limit=-1 bin/magento migrate:delta --reset -a /magento/vendor/magento/data-migration-tool/etc/opensource-to-opensource/1.9.3.8/config.xml
Reset the current position of migration to start from the beginning

[2018-08-09 13:11:49][INFO][mode: delta][stage: delta delivering][step: Customer Attributes Step]: started
[2018-08-09 13:11:49][INFO][mode: delta][stage: volume check][step: Customer Attributes Step]: started
16% [====>-----------------------] Remaining Time: < 1 sec
[2018-08-09 13:11:49][ERROR]: Attribute vat_id does not exist in the type customer_address
[2018-08-09 13:11:49][WARNING]: Volume Check failed
[2018-08-09 13:11:49][INFO][mode: delta][stage: delta delivering][step: Map Step]: started
[2018-08-09 13:11:49][INFO][mode: delta][stage: volume check][step: Map Step]: started
100% [============================] Remaining Time: < 1 sec

thanks in advance for your help on this. 
I tried reindexing, purging the cache but no effect. Even reinstalling a fresh site did not work. 

Bonus, I have no table called sequence_.. and do not know how to proceed if i can fix from mysql

**Comment by victor-v-rad** [comment]
Hi @jmminy 

sequence_* tables are created in \Migration\Step\SalesIncrement\Data . Run this step separately to debug why it do not create these tables

**Comment by victor-v-rad** [comment]
Hi @jmminy 

Sorry for late reply. Did you fix it? You can remove OrderGrids Step from migration and run reindex in M2 to fill the grid tables. As for sequence_* tables. The tool creates it during migration. So recheck your MySQL connection has necessary privileges

**Comment by jmminy** [comment]
Hi. Nope I did not have any chance to do so. Removing ordergrid step will prevent order from being saved to M2 right ? This is not an option unfortunately 😔

**Comment by jmminy** [comment]
And both users have full permissions in mysql

**Comment by victor-v-rad** [comment]
>will prevent order from being saved to M2

OrderGrids Step just fills grid tables in M2 and orders are migrated in different steps. Orders will be in M2 after migration, and , grid tables can be regenerated after reindex.

**Comment by jmminy** [comment]
Hi, 
@victor-v-rad thanks for your help and sorry for the late reply. I had to test and re-test again after validating env again. 

I did fix the issue by running the steps separately. It appears that my issue was caused by different factors: 

1- I have multiple stores with different store views. All where created in the M2 instance but i had to manually add the sequence_* tables . this topic helped a lot too https://github.com/magento/magento2/issues/12318
2- After that I could see the orders in DB but not in the grid. To fix the grid issue, I had to reindex and recompile magento. After that everything was ok.

To finish, now I would just know how to remove properly useless tables after migration as I do not find this point in the documentation. 
thanks a lot for your help you gave me good paths to debug.
